test/addons/symlinked-module/test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../../common');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+// This test verifies that symlinked native modules can be required multiple
+// times without error. The symlinked module and the non-symlinked module
+// should be the same instance. This expectation was not previously being
+// tested and ended up being broken by https://github.com/nodejs/node/pull/5950.
+
+// This test should pass in Node.js v4 and v5. This test will pass in Node.js
+// with https://github.com/nodejs/node/pull/5950 reverted.
+
+common.refreshTmpDir();
+
+const addonPath = path.join(__dirname, 'build', 'Release');
+const addonLink = path.join(common.tmpDir, 'addon');
+
+try {
+  fs.symlinkSync(addonPath, addonLink);
+} catch (err) {
+  if (err.code !== 'EPERM') throw err;
+  console.log('1..0 # Skipped: module identity test (no privs for symlinks)');
```

Comment:
you can use `common.skip` for this now... just landed in https://github.com/nodejs/node/commit/52bae222a3a8480b2b30d8dc4629b64a284410d4